app/models/user.rb

```diff
@@ -1500,6 +1501,23 @@ def display_name
     end
   end
 
+  def clear_status
+    user_status.destroy if user_status
+  end
+
+  def set_status(description)
```

Comment:
```suggestion
  def set_status!(description)
```

Minor suggestion to reflect that this method raises an exception if it fails